src/main/java/seedu/address/model/expense/ExpenseList.java

```diff
@@ -140,7 +140,7 @@ public boolean equals(Object other) {
      */
     public boolean contains(Expense toCheck) {
         requireNonNull(toCheck);
-        return internalListOfExpenses.stream().anyMatch(toCheck::isSameExpense);
+        return internalListOfExpenses.stream().anyMatch(toCheck::equals);
```

Comment:
Good to declutter the codebase for expenses by using overriding equals method instead of defining extra methods.